pom.xml

```diff
@@ -278,16 +225,17 @@
             <version>4.0.10</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>javax.xml.bind</groupId>
-            <artifactId>jaxb-api</artifactId>
-            <version>2.3.1</version>
-        </dependency>
         <dependency>
             <groupId>io.swagger.core.v3</groupId>
-            <artifactId>swagger-annotations</artifactId>
+            <artifactId>swagger-core-jakarta</artifactId>
             <version>${swagger-version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.parsson</groupId>
+            <artifactId>parsson</artifactId>
+            <version>1.1.7</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
What is this and why do we need it?

Reply:
It's used in PublicApiStub (test) by jakarta.json.JsonObjectBuilder as default implementation of JsonProvider. It does not come bundled in jakarta.json-api in case you want to use a different implementation.